src/med_bench/nuisances/cross_conditional_outcome.py

```diff
@@ -0,0 +1,224 @@
+"""
+the objective of this script is to provide nuisance estimators 
+for mediation in causal inference
+"""
+
+import numpy as np
+
+from sklearn.base import clone
+
+from med_bench.utils.utils import _get_train_test_lists, _get_interactions
+
+def estimate_cross_conditional_mean_outcome_discrete(m, x, y, f, regressors):
```

Comment:
not sure about a separate file from `estimate_conditional_mean_outcome`

Reply:
Indeed, it would be fine having all  nuisances in one module IMHO